crp_APL/controllers/ctrl_vehicle_control_lat_stanley/launch/ctrl_vehicle_control_lat_stanley.launch.py

```diff
@@ -0,0 +1,18 @@
+from launch import LaunchDescription
+from launch_ros.actions import Node
+
+def generate_launch_description():
+    ld = LaunchDescription()
+
+    ctrl_vehicle_control_lat_stanley = Node(
+        package="ctrl_vehicle_control_lat_stanley",
+        executable="ctrl_vehicle_control_lat_stanley",
+        parameters=[
+            {"/ctrl/k_gain": 0.5},
+            {"/ctrl/wheelbase": 2.7},
```

Comment:
This should be a parameter. Probably in an other task we should sync vehicle params as global (as in crp repo) parameters in every package.

Reply:
ye should make another task for it